applications/nrf_desktop/board_configuration.rst

```diff
@@ -103,18 +103,18 @@ Sample dongle (``nrf5340dk/nrf5340``)
         Input data comes from Bluetooth and is retransmitted to USB.
       * The configuration with the B0 bootloader is set as default.
 
-Sample mouse or keyboard (``nrf54l15pdk/nrf54l15/cpuapp``)
-      * The configuration uses the nRF54L15 Preview Development Kit (PDK).
+Sample mouse or keyboard (``nrf54l15dk/nrf54l15/cpuapp``)
+      * The configuration uses the nRF54L15 Development Kit (DK).
       * The build types allow to build the application as a mouse or a keyboard.
       * Inputs are simulated based on the hardware button presses.
       * On the nRF54L15 SoC, you can only use the **GPIO1** port for PWM hardware peripheral output.
-        Because of that, on the PDK PCA10156 revision v0.3.0, **LED 0** and **LED 2** cannot be used for PWM output.
+        Because of that, on the PDK PCA10156 revision v0.8.1, **LED 0** and **LED 2** cannot be used for PWM output.
```

Comment:
```suggestion
        Because of that, on the DK PCA10156 revision v0.8.1, **LED 0** and **LED 2** cannot be used for PWM output.
```
Should this be a DK as well?

Reply:
Yes, I overlooked it. Thanks !